.github/workflows/main-trigger-people-deploy.yml

```diff
@@ -1,5 +1,15 @@
 name: Trigger-People-Build
 on:
+  push:
+    branches:
+    - main
+    paths-ignore:
+    # Ignore files that have no impact on the build
+    - '.github/assign-by-files.yml'
+    - '.github/pull_request_template.md'
+    - '.github/instructions/*'
+    - '_docs/*'
+    - 'README.md'
   workflow_dispatch:
```

Comment:
I think you need to add workflow_call as a trigger

https://docs.github.com/en/actions/using-workflows/reusing-workflows#creating-a-reusable-workflow

Reply:
Done and added additional comments to make sense why there are 3 different triggers,